source/welcome/log-out.rst

```diff
@@ -17,4 +17,22 @@ You can log out of Mattermost from your profile picture. Select **Log Out** to l
 
       .. image:: ../images/profile-log-out-mobile.png
         :width: 300
-        :alt: Log out of Mattermost from your profile picture.
\ No newline at end of file
+        :alt: Log out of Mattermost from your profile picture.
+
+Frequently asked questions
+--------------------------
+
+What happens when I log out of Mattermost?
```

Comment:
I'm pretty sure most of these only apply to mobile. I can't 100% speak for the desktop app, but most of these things don't exist in the web app or aren't stored in a way that lasts through a refresh